_build/build_gum.py

```diff
@@ -179,8 +245,14 @@ def setup_directories(gum_source, gum_target):
 		sys.exit()
 
 	# Inject gum_target in pepper_params and replace os.sep with URI slash
-	pepper_params = pepper_params.replace("**gum_tmp**",os.path.abspath(pepper_tmp).replace(os.sep,"/"))
-	pepper_params = pepper_params.replace("**gum_target**",gum_target.replace(os.sep,"/"))
+	#if platform.system() == "Windows":
+	#	pepper_params = pepper_params.replace("**gum_tmp**",os.path.abspath(pepper_tmp).replace(os.sep,"/"))
+	#	pepper_params = pepper_params.replace("**gum_target**",gum_target.replace(os.sep,"/"))
+	#else:
+	pepper_params = pepper_params.replace("file:/**gum_tmp**", os.path.abspath(pepper_tmp))
+	pepper_params = pepper_params.replace("file:/**gum_target**", os.path.abspath(pepper_home) + os.sep + "../../target/")
+	pepper_params = pepper_params.replace("file:/**gum_home**", os.path.abspath(pepper_home) + os.sep + "../../../")
```

Comment:
@nitinvwaran Here too we want PAULA and ANNIS data to be generated in `target/annis/` and `target/paula/` (this is later zipped and placed in root level `paula/` to avoid tracking huge numbers of files in git)